platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/metrics/EventIntakeMetrics.java

```diff
@@ -68,44 +69,68 @@ public class EventIntakeMetrics implements StaleEventObserver {
             new RunningAverageMetric.Config(INTERNAL_CATEGORY, "shouldCreateEvent").withFormat(FORMAT_10_1);
     private final RunningAverageMetric shouldCreateEvent;
 
-    private static final Counter.Config STALE_EVENTS_TOTAL_CONFIG =
-            new Counter.Config(INTERNAL_CATEGORY, "staleEvTot").withDescription("total number of stale events ever");
-    private final Counter staleEventsTotal;
-
     private static final SpeedometerMetric.Config STALE_EVENTS_PER_SECOND_CONFIG = new SpeedometerMetric.Config(
-                    INTERNAL_CATEGORY, "staleEv/sec")
-            .withDescription("number of stale events per second")
-            .withFormat(FORMAT_16_2);
+                    INTERNAL_CATEGORY, "staleEventsPerSecond")
+            .withDescription("number of stale events per second");
     private final SpeedometerMetric staleEventsPerSecond;
 
+    private static final SpeedometerMetric.Config STALE_SELF_EVENTS_PER_SECOND_CONFIG = new SpeedometerMetric.Config(
+                    INTERNAL_CATEGORY, "staleSelfEventsPerSecond")
+            .withDescription("number of stale self events per second");
+    private final SpeedometerMetric staleSelfEventsPerSecond;
+
+    private static final SpeedometerMetric.Config STALE_APP_TRANSACTIONS_PER_SECOND_CONFIG =
```

Comment:
I think these metrics will be better as counters. It is simple to render them as changes (spikes) in Grafana instead of just an ever-increasing maximum. I suggest this because I think the counters are more performant that the speedometer metric, and we plan on eliminating it at some point.